UnityPy/classes/AnimationClip.py

```diff
@@ -136,7 +136,7 @@ def UnpackInts(self):
         for i in range(self.m_NumItems):
             bits = 0
             entry = 0
-            while bits << m_BitSize:
+            while bits < m_BitSize:
```

Comment:
Have you investigated this?

AssetStudio also uses <.
[AnimationClip - line 112](https://github.com/Perfare/AssetStudio/blob/dfb74baf79cf6edd746c1e4f976e3bb03ff88876/AssetStudio/Classes/AnimationClip.cs#L112)

Reply:
Misread it ？ This fix is from << to <

Reply:
Oh, sorry, you're right.